29,33d28
< export enum BOT_TYPES {
<     Messenger = "Messenger",
<     Frontend = "Frontend",
<   }
< 
361c356
< 					send(psid, "Failed to- send a chuck ðŸ˜¢")
---
> 					send(psid, "Failed to send a chuck ðŸ˜¢")

--- main.ts	2024-08-07 20:34:17.764474159 +0800
+++ /data/data/com.termux/files/home/github/GGMC/src/main.ts	2024-08-07 12:05:59.142514097 +0800
@@ -26,11 +26,6 @@
 export const VERSION: string = "1.5.5";
 export const upStartTime: number = Date.now();
 
-export enum BOT_TYPES {
-    Messenger = "Messenger",
-    Frontend = "Frontend",
-  }
-
 interface MainRuntileUtils {
     isDevRunning: boolean;
     apiUrl: string;
@@ -358,7 +353,7 @@
     return new Promise((resolve,reject) => {
 				send(psid, i, true)
 				.catch(e=>{
-					send(psid, "Failed to- send a chuck ðŸ˜¢")
+					send(psid, "Failed to send a chuck ðŸ˜¢")
 						.catch(e=>{
 							console.log("Something wrong when posting a message?")
 							return reject()
